proxymode/proxymode.go

```diff
@@ -79,7 +112,7 @@ func (pm *ProxyMode) Search(r *http.Request) (packages.Packages, error) {
 	proxyURL.Scheme = pm.destinationURL.Scheme
 	proxyURL.User = pm.destinationURL.User
 
-	proxyRequest, err := http.NewRequest(http.MethodGet, proxyURL.String(), nil)
+	proxyRequest, err := retryablehttp.NewRequest(http.MethodGet, proxyURL.String(), nil)
```

Comment:
Do requests also need to be retryable?

Reply:
Yes, as the client demands a different structure (`retryablehttp.Request`).